Libraries/LibGUI/Clipboard.cpp

```diff
@@ -62,6 +64,25 @@ Clipboard::DataAndType Clipboard::data_and_type() const
     return { data, type };
 }
 
+AK::RefPtr<Gfx::Bitmap> Clipboard::get_bitmap(Gfx::BitmapFormat format, Gfx::Size size)
+{
+    auto response = WindowServerConnection::the().send_sync<Messages::WindowServer::GetClipboardContents>();
+    if (response->shbuf_id() < 0)
+        return {};
+    auto shared_buffer = SharedBuffer::create_from_shbuf_id(response->shbuf_id());
+
+    if (!shared_buffer) {
+        dbgprintf("GUI::Clipboard::data() failed to attach to the shared buffer\n");
+        return {};
+    }
+    if (response->content_size() > shared_buffer->size()) {
+        dbgprintf("GUI::Clipboard::data() clipping contents size is greater than shared buffer size\n");
+        return {};
+    }
+    auto bitmap = Gfx::Bitmap::create_with_shared_buffer(format, *shared_buffer, size);
+    return bitmap;
```

Comment:
No need for a temporary here:
```suggestion
    return Gfx::Bitmap::create_with_shared_buffer(format, *shared_buffer, size);
```